app/models/post.rb

```diff
@@ -33,6 +33,10 @@ class Post < ApplicationRecord
     self.touch(:interacted_at)
   end
 
+  before_destroy do
+    reshares.each {|reshare| reshare.update_attributes(root_guid: nil) }
```

Comment:
This ought to be possible with a single update statement, if reshares is a relation just skipping the each should do it.

Reply:
No, I could use `update_all` (like in the migration):
```ruby
reshares.update_all(root_guid: nil)
```
but pronto doesn't like this: ``app/models/post.rb:37 W: Avoid using `update_all` because it skips validations.``

Should I disable pronto on this line too?

Reply:
I think so, it's a innocuous enough statement, and doing an N+1 query here just feels worse to me.

Reply:
You're right, after thinking more about it, it makes much more sense that way. I updated it.